File: ui/src/classes/Package.ts
import Ticket from "@/classes/Ticket";
import Person from "@/classes/Person";
import Room from "@/classes/Room";
import Building from "./Building";
import Address from "./Address";
import City from "./City";

export default class Package {
    public id: string = "";
    public receiver: Person = new Person("", "", "");
    public collectionPoint: Room = new Room("", "", new Building("","", new Address(new City("",""),"","", 0,"")));
    public sender: string = "";
    public name: string = "";
    public status: string = "";
    public routeFinished: boolean = false;
    public tickets: Array<Ticket> = [];
}

File: pakketservice/Database/Datamodels/Package.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PakketService.Database.Datamodels
{
    public class Package
    {
        [Key]
        public Guid Id { get; set; }
        public string ReceiverId { get; set; }
        public Guid CollectionPointId { get; set; }
        public string Sender { get; set; }
        public string Name { get; set; }
        public Status Status { get; set; }
        public bool RouteFinished { get; set; }
        public virtual ICollection<Ticket> Tickets { get; set; }
    }
}
